client/gutenberg/extensions/todo/editor.scss

```diff
@@ -14,7 +16,7 @@ $light-gray-900: #a2aab2;
 $light-gray-800: #b5bcc2;
 $light-gray-700: #ccd0d4;
 $light-gray-600: #d7dade;
-$light-gray-500: #e2e4e7;	// good for "grayed" items and borders
+$light-gray-500: #e2e4e7; // good for "grayed" items and borders
 $light-gray-400: #e8eaeb;
 $light-gray-300: #edeff0;
 $light-gray-200: #f3f4f5;
```

Comment:
Looks like only 200, 500 and 700 from light-greys and 200 and 500 from dark-grey are used at the moment. Rest of these variables are unused.

I [tested](https://github.com/Automattic/wp-calypso/pull/26777/files#diff-90e4db7a1b7a4bd67a15a28128676026) simply to import these from Gutenberg but that didn't seem right. 🤔

Reply:
This list clearly doesn't belong here, but let's leave that out of scope for this PR.